src/backend/embedding/extract_file/extract_file.py

```diff
@@ -239,9 +250,163 @@ def extract_pdf_content(self, customer_guid: str, file_path: str, filename: str)
             logger.error(f"Error extracting content from PDF file '{filename}': {e}")
             raise Exception(f"PDF content extraction failed: {e}")
 
+    def extract_docx_content(self, customer_guid: str, file_path: str, filename: str):
+        try:
+            results = []  # This should store the page contents
+            doc = Document(file_path)
+
+            # Define maximum number of paragraphs per page for simulation
+            max_paragraphs_per_page = 20  # Adjust this based on your document layout
```

Comment:
What is the need for this variable max_paragraphs_per_page
I dont understand the significance of this variable and why we are hardcoding it to 20?

Reply:
The variable max_paragraphs_per_page is used in the code to simulate page boundaries within a Microsoft Word document (.docx). This is necessary because .docx files do not natively store information about where one page ends and the next begins.
Many Word documents may have around 20 paragraphs per page on average, so it serves as a reasonable default sir.

Reply:
Can we extract documents without using max_paragraphs_per_page 

The problem here is we would be uploading thousands of documents and we cannot hard code these variables